plugins/qtAccessionPlugin/lib/model/QubitAccession.php

```diff
@@ -103,7 +103,9 @@ public static function nextAvailableIdentifier()
       $con->beginTransaction();
 
       // Determine what should be the next identifier
-      $identifier = Qubit::generateIdentifierFromCounterAndMask(self::nextAccessionNumber(), sfConfig::get('app_accession_mask'));
+      $setting = QubitSetting::getByName('accession_mask');
+      $mask = $setting->getValue(array('sourceCulture' => true));
+      $identifier = Qubit::generateIdentifierFromCounterAndMask(self::nextAccessionNumber(), $mask);
```

Comment:
Why do you need to do this? sfConfig should be populated with the settings in CLI tasks if they extend arBaseTask and I believe csvAccessionImportTask inherits from it through csvImportBaseTask.

Reply:
Oh, I see. We're not calling the `parent::execute` from this import task, like we do in:

https://github.com/artefactual/atom/blob/6ad917404acd4f2a8c53f90c034b99d4fb2312e1/lib/task/import/csvImportTask.class.php#L138

If you could add it and keep using sfConfig in here to maintain consistency, that would be great.

Reply:
Ah! Good to know! I'll change that.